### ISSUE TITLE
Releases or Binaries

### ISSUE THREAD

**Issue by ashes999** [sq/JSIL#1001]
Hi,

TLDR: would you reconsider your policy on not having tagged releases or binaries of JSIL?

I find it frustrating to rebuild from source each time, and that I don't always get the same version when I clone on two different machines (unless I memorize which commit I checked out).

I believe the last time this was brought up, your worry was that people will perceive this as some sort of "stable" release and expect something (in terms of quality) that you can't provide.

Would you be open to either:

- Arbitrarily tagging weekly or monthly snapshots as such, under "releases"?
- Providing (or someone else providing) pre-built binaries so that JSIL is easier to consume?

I can volunteer to do the latter, provided I can get this to build on at least one of my machines.

**Comment by iskiselev** [comment]
I plan to create public binary releases for JSIL, but there are some works to be done priory it could be automated:
- Copy JS Libraries, compatible with translator on translation.
- Understand how to mark unique semantic version for each commit (or at least release commits)
- Configure building on myget (it should check test results priory on Travis/Appveyor probably? or it should be just artifact publishing?)

I can't promise when it will be done :( If anybody will help - it will be sooner (if @kg has no objectives).
